eslint/eslintrc.yml

```diff
@@ -23,15 +23,21 @@ globals:
 
 extends:
     # Extend the base Body Labs style, which is pulled in via peerDependencies.
-    "../../bodylabs-javascript-style/eslint/eslintrc-shared.yml"
+    - '../../bodylabs-javascript-style/eslint/eslintrc-shared.yml'
 
-rules:
-    # 2 means error, 1 means warning, 0 means ignore.
+    # Pull in https://github.com/yannickcr/eslint-plugin-react
+    - 'plugin:react/recommended'
 
+rules:
     # JSX.
-    # eslint-plugin-react is required
-    # https://github.com/yannickcr/eslint-plugin-react
-    jsx-quotes: 2
+    jsx-quotes: 'error'
+
+    # More JSX, from https://github.com/yannickcr/eslint-plugin-react
+    react/jsx-curly-spacing: ['error', 'always']
+    react/jsx-equals-spacing: ['error', 'never']
+    react/jsx-indent: 'error'
+    react/jsx-pascal-case: 'error'
+    react/jsx-space-before-closing: 'error'
 
```

Comment:
We are pulling in the [recommended configuration](https://github.com/yannickcr/eslint-plugin-react#recommended-configuration) which includes jsx-no-duplicate-props.

Totally fine with jsx-key, but do you want to try it out in creel-frontend and make sure it doesn't generate false positives first?

jsx-sort-props I'm not sure about. As a nuanced person, I tend to think that a developer's chosen order may be more useful than an alphabetical order.